bin/qgis/smoderp2d-plugin/smoderp_2D_dockwidget.py

```diff
@@ -580,20 +581,27 @@ def _layerColorRamp(layer):
     def importResults(self):
         """Import results into QGIS, group them and show them as layers.
         """
-        def import_group_layers(group, outdir, ext='asc', show=False):
+        def import_group_layers(group, outdir, ext=['asc', 'gml'], show=False):
```

Comment:
Is it necessary to have a list here? I suggest having a tuple is a better idea unless you really need a list because mutable types as default values can cause big headaches.

Reply:
Right, changed in 0d915c66.